src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
```

Comment:
Use decomposition structure whererver it`s possible
```suggestion
export const Person = ({ person }) => {

const { name, age, ...} = person;
```

OR
```suggestion
export const Person = ({ person: { name, ... } }) => {
```